pkgs/shells/marcel/default.nix

```
@@ -0,0 +1,56 @@
+{ lib
+, stdenv
+, buildPythonPackage
+, fetchFromGitHub
+, setuptools
+, dill
+, psutil
+, makeWrapper
+, python3
+, bash
+}:
```

```suggestion
{ lib
, buildPythonPackage
, fetchFromGitHub
, dill
, psutil
, python3
, bash
}:
```
There are some unused imports.
(`makeWrapper` is provided by `buildPythonPackage`)